database/ticket_test.go

```diff
@@ -67,14 +78,6 @@ func testInsertNewTicket(t *testing.T) {
 		t.Fatal("expected an error inserting ticket with duplicate hash")
 	}
 
-	// Inserting a ticket with the same fee address should fail.
-	ticket3 := exampleTicket()
-	ticket3.FeeAddress = ticket.FeeAddress
-	err = db.InsertNewTicket(ticket3)
-	if err == nil {
-		t.Fatal("expected an error inserting ticket with duplicate fee addr")
-	}
-
```

Comment:
Not sure why this was removed, `testHasFeeAddress` isn't checking for persisting a ticket with duplicate a duplicate fee address.

Reply:
I removed it from that function, because if this endpoint goes in, it will create a new path that does not need to check the fee address, because it doesn't put one, and then if it's already in the db but with no fee address, this will need to be checked when the first fee address goes in, so outside of insert, during an update.

This also brings up the question, why aren't we checking **every** fee address that is created? imo the check isn't needed at all, but why only on inserts? A new address can also be added during update in master now. It seems the "risk" of inserting a duplicate address is the same for either insert or update.